include/solvers/fluid_dynamics_matrix_free_operators.h

```diff
@@ -493,6 +571,23 @@ class NavierStokesOperatorBase : public Subscriptor
   Table<2, Tensor<1, dim + 1, VectorizedArray<number>>>
     time_derivatives_previous_solutions;
 
+  /**
+   * @brief Table with correct alignment for vectorization to store the values
+   * of the previous newton step at the faces.
+   *
```

Comment:
```suggestion
```

Reply:
All of the other briefs are currently in the same way in that file, so I'll keep it uniform for now.